### PR TITLE
feat(awesome: main: check): print config path, and do it to stdout

### PR DESCRIPTION
**Pull request by actionless**
(follow-up on @blueyed comment here: https://github.com/awesomeWM/awesome/pull/2687#issuecomment-540129901)

### CODE REVIEW

File: awesome.c [hunk 1]
```diff
@@ -700,12 +701,12 @@ main(int argc, char **argv)
 
         if(!success)
         {
-            fprintf(stderr, "✘ Configuration file syntax error.\n");
+            fprintf(stdout, "ERROR\n");
```
Comment (blueyed) [inline awesome.c]: I think "ERROR: " could be used as a prefix with the error above already.
Then it would only return here.

Comment (actionless) [reply awesome.c]: could you insert is as a "Suggestion" here?
i'm not sure i'm following

Comment (blueyed) [reply awesome.c]: Remove the line above, and add "ERROR: " in front of `fprintf(stderr, "%s\n", err);` (I cannot make a suggestion there).
=> `fprintf(stderr, "ERROR: %s\n", err);`

Comment (actionless) [reply awesome.c]: do you mean like this?

**Comment by psychon** [comment]
I just tried a random non-Lua file with the current code:
```
$ awesome -c  /tmp/b -k
/tmp/b:1: syntax error near 'MIN'
✘ Configuration file syntax error.
```
So, after this PR, the output would be... uhm, I guess this:
```
$ awesome -c  /tmp/b -k
Checking config /tmp/b... ERROR: /tmp/b:1: syntax error near 'MIN'
```
Right?

And for a working file, I would get:
```
Checking config /tmp/b... OK
```
Fine with me, It's just so that we know what is going on.

**Comment by blueyed** [comment]
Yes.
Maybe the "ERROR: …" could be on a new line (if you want to parse it etc)?

**Comment by actionless** [comment]
updated

File: awesome.c [hunk 2]
```diff
@@ -686,6 +686,7 @@ main(int argc, char **argv)
         bool success = true;
         /* Get the first config that will be tried */
         const char *config = luaA_find_config(&xdg, confpath, true_config_callback);
+        fprintf(stdout, "Checking config %s... ", config);
```
Comment (psychon) [inline awesome.c]:
```suggestion
        fprintf(stdout, "Checking config '%s'... ", config);
```
(No idea if this makes sense or not)

**Review by Elv13 — DISMISSED** [review]

**Review by psychon — DISMISSED** [review]

**Comment by actionless** [comment]
this one will require manual merge as too much old commits are failing in "test prev commits" travis scenario

**Comment by actionless** [comment]
bump

**Review by Aire-One — APPROVED** [review]

**Comment by actionless** [comment]
this time, codecov stucked :facepalm:

**Review by sclu1034 — APPROVED** [review]

**Review by Elv13 — APPROVED** [review]

**Pull request merged**